components/list-issues.tsx

```diff
@@ -291,14 +293,14 @@ export default function ListIssues({
       !loading.isLoading ? (
         <div className="pt-4">
           <NothingFound description={emptyMessage || filterByState.emptyState}>
-            <ReadOnlyButtonWrapper>
-              <InternalLink
-                className="read-only-button"
-                label={buttonMessage || String(t("actions.create-one"))}
-                href={redirect || getURLWithNetwork("/create-bounty")}
-                uppercase
-              />
-            </ReadOnlyButtonWrapper>
+            {wallet?.address && (
+              <>
+                <Button onClick={() => setShowCreateBounty(true)}>
+                  {buttonMessage || String(t("actions.create-one"))}
+                </Button>
+                <CreateBountyModal show={showCreateBounty} setShow={setShowCreateBounty}/>
```

Comment:
Shouldn't the create modal bounty be on a higher level component ? (like we spoke)